test/setup.js

```diff
@@ -168,3 +186,13 @@ function extractImportantFields (parsedTorrent) {
   const priv = parsedTorrent.private // private is a reserved word in JS
   return { infoHash, name, announce, urlList, comment, 'private': priv }
 }
+
+function copy (pathFrom, pathTo) {
+  try {
+    fs.copySync(pathFrom, pathTo)
+  } catch (e) {
+    // Windows sometimes gives us an EPERM error even tho the copy happened...
+    if (process.platform !== 'win32' || e.code !== 'EPERM') throw e
+    console.log('ignoring windows copy EPERM error', e)
```

Comment:
Turns out the copy is actually **not** working. This pretty much just skips that one part of the test on Windows.

There is a bug either in node or `fs-extra`. Windows lets the test process create files and folders underneath `C:\Windows\Temp`, but when you try `fs.copySync` to that location you get this error.

The path that we're copying doesn't exist yet, so definitely no open file handles. This is also the path that `fs.copySync` is trying to open under the hood.